code/modules/antagonists/fugitive/hunter.dm

```diff
@@ -28,10 +28,14 @@
 			to_chat(owner, "<B>The criminals should be on the station, we have special huds implanted to recognize them.</B>")
 			to_chat(owner, "<B>As we have lost pretty much all power over these damned lawless megacorporations, it's a mystery if their security will cooperate with us.</B>")
 		if("russian")
-			to_chat(src, span_danger("Ay blyat. I am a space-russian smuggler! We were mid-flight when our cargo was beamed off our ship!"))
-			to_chat(src, span_danger("We were hailed by a man in a green uniform, promising the safe return of our goods in exchange for a favor:"))
-			to_chat(src, span_danger("There is a local station housing fugitives that the man is after, he wants them returned; dead or alive."))
-			to_chat(src, span_danger("We will not be able to make ends meet without our cargo, so we must do as he says and capture them."))
+			to_chat(owner, span_danger("Ay blyat. I am a space-russian smuggler! We were mid-flight when our cargo was beamed off our ship!"))
+			to_chat(owner, span_danger("We were hailed by a man in a green uniform, promising the safe return of our goods in exchange for a favor:"))
+			to_chat(owner, span_danger("There is a local station housing fugitives that the man is after, he wants them returned; dead or alive."))
+			to_chat(owner, span_danger("We will not be able to make ends meet without our cargo, so we must do as he says and capture them."))
+		if("bounty hunters")
```

Comment:
these should really be defines local to this file or tied to defines elsewhere but you don't need to pass that in this PR